console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -2128,23 +2128,48 @@ createIotDevice("iotProjectFrance", {
 
 createIotDevice("iotProjectFrance", {
   deviceId: "12",
-  jsonData: JSON.stringify({ via: 11, ext: { summit: "Mt Blanc" } }),
+  jsonData: JSON.stringify({
+    via: ["device-1", "device-2"],
+    ext: { summit: "Mt Blanc" }
+  }),
   credentials: []
 });
 
 createIotDevice("iotProjectIndia", {
   deviceId: "20",
   enabled: true,
   viaGateway: false,
-  jsonData: JSON.stringify({ ext: { ocean: "indian" } }),
+  jsonData: JSON.stringify({
+    via: ["20", "21"],
+    default: {
+      "content-type-1": "text/plain",
+      "content-type-2": "text/plain",
+      "content-type-3": "text/plain",
+      long: 12.3544
+    },
+    ext: {
+      custom: {
+        level: 0,
+        serial_id: "0000",
+        location: {
+          long: 1.234,
+          lat: 5.678
+        },
+        features: ["foo", "bar", "baz"]
+      }
+    }
+  }),
   credentials: []
 });
 
 createIotDevice("iotProjectIndia", {
   deviceId: "21",
   enabled: true,
   viaGateway: true,
-  jsonData: JSON.stringify({ via: 20, ext: { summit: "Kanchenjunga" } }),
+  jsonData: JSON.stringify({
+    via: ["device-3", "device-4"],
```

Comment:
same as above. Either remove the via devices or set `viaGateway` to false.

Reply:
yes, I made changes you suggested.